src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -3155,7 +3155,7 @@ def test_network_watcher_vm(self, resource_group, resource_group_location):
         self.cmd('network watcher show-next-hop -g {rg} --vm {vm} --source-ip 123.4.5.6 --dest-ip 10.0.0.6')
 
     @ResourceGroupPreparer(name_prefix='cli_test_nw_flow_log', location='eastasia')
-    @StorageAccountPreparer(name_prefix='clitestnw', location='eastasia')
+    @StorageAccountPreparer(name_prefix='clitestnw', location='eastasia', kind='StorageV2')
```

Comment:
```
azure.mgmt.network.v2019_11_01.models._models_py3.ErrorResponseException: (UnsupportedStorageAccountVersionWithRetention) Storage /subscripti
ons/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/cli_test_nw_flow_logfrumyb6ki53obj262ik24kxqlqatgydxxqj3sgia37ogd7jlcqxxdjq/providers/Microsoft.S
torage/storageAccounts/clitestnwenwnk7nks5dumov is a V1 storage account and with retention policy. Only V2 storage account are supported with retention p
olicy. Read more -> aka.ms/LogsHelp
```
Out of dated way to configure flow-log of watcher. There must be changes on service to restrict only V2 to configure a flow-log instead the original V1.